pyme-depends/meta.yaml

```diff
@@ -128,15 +128,18 @@ requirements:
   run:
     - python {{ python }}
     - numpy >=1.11
+    - numpy >= 1.4 [py3k]

```

Comment:
why do we need a higher numpy on py3?

Reply:
I thought you'd be more receptive to letting us upgrade numpy on py3. Ideally we'd pin to 1.4 or higher on py2 as well.